harvester/periodic-jobs/src/main/java/dk/dbc/dataio/harvester/periodicjobs/DailyProofingHarvestOperation.java

```diff
@@ -60,6 +60,7 @@ RecordFetcher getRecordFetcher(RecordIdDTO recordId, RecordServiceConnector reco
         return new RecordFetcher(recordId, recordServiceConnector, config);
     }
 
+    @SuppressWarnings("PMD.UnnecessaryFullyQualifiedName")
```

Comment:
Og det er jo på en måde fjollet at de så hedder det samme.

Reply:
Enig. Jeg kunne ikke regne ud hvad det var meningen at getter funktionen skulle returnere, derfor turde jeg ikke omdøbe noget.